ipalib/constants.py

```diff
@@ -374,3 +374,14 @@
 }
 
 ALLOWED_NETBIOS_CHARS = string.ascii_uppercase + string.digits + '-'
+
+# vault data wrapping algorithms
+VAULT_WRAPPING_3DES = 'des-ede3-cbc'
+VAULT_WRAPPING_AES128_CBC = 'aes-128-cbc'
+VAULT_WRAPPING_SUPPORTED_ALGOS = (
+    # 3DES for backwards compatibility
+    VAULT_WRAPPING_3DES,
```

Comment:
I recommend that you only include 3DES if it is actually supported. This will give a clean error message instead of failing with an internal error:

```
from cryptography.hazmat.primitives.ciphers import algorithms, modes

backend.cipher_supported(algorithms.TripleDES(b"\x00" * 8), modes.CBC(b"\x00" * 8))
```